src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
I'm not sure we want such opaque enum values. Why can't we be more explicit and add enum values for specific client and server types as we need them?
e.g.  we can add JAVA_ASYNC_CLIENT_WITH_CORKING (don't think it exists just an example)  once we have such java client and the resulting name is much more descriptive. Then we can use the same enum values to store results in bigquery (because they are descriptive).

Over time, if we stop testing some client types and start testing new ones, we can just add new enum values and deprecate the old ones which will end up more comprehensible than reusing the CLIENT_VARIANT_X values - and we won't risk messing up are existing measurements.